application/application.go

```diff
@@ -0,0 +1,166 @@
+package application
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/config"
+	"github.com/josephroberts/edge-node-manager/database"
+	"github.com/josephroberts/edge-node-manager/device"
+	"github.com/josephroberts/edge-node-manager/helper"
+	"github.com/josephroberts/edge-node-manager/proxyvisor"
+	"github.com/josephroberts/edge-node-manager/radio"
+)
+
+type ApplicationInterface interface {
+	Process() error
+	loadDevices() (map[string]device.DeviceInterface, error)
```

Comment:
Is it useful to have unexported methods here? Will they only be used in the `Application` package? Since an interface is describing what a struct needs to implement it seems a bit questionable to have these here, unless I'm missing something?